pkg/tso/allocator_manager.go

```diff
@@ -314,10 +314,6 @@ func CalSuffixBits(maxSuffix int32) int {
 func (am *AllocatorManager) SetUpAllocator(parentCtx context.Context, dcLocation string, leadership *election.Leadership) {
 	am.mu.Lock()
 	defer am.mu.Unlock()
-	if am.updatePhysicalInterval != defaultTSOUpdatePhysicalInterval {
```

Comment:
Why remote this part?

Reply:
Shouldn't this sanity check happen once server config cfg.TSOUpdatePhysicalInterval be initialized? After we moved allocator_manager.go from server/tso package to pkg/tso, we have to define maintain a copy of defaultTSOUpdatePhysicalInterval which could diverge from the original one.

BTW, this check and warning is weird, it must have some unknown context that I don't know. I'll add it back and leave the optimization in the future.

Reply:
@JmPotato , would this be good enough for you if I move this sanity check to server/config.go immediately after it's initialized?

Reply:
BTW, I have restored the check. The above question is still valid.

Reply:
It's ok for this check to be moved around as long as it's kept. Because `updatePhysicalInterval` is a very internal config and may cause some undefined behavior if it's changed to some unreasonable value, this check is meant to warn the one who changed it.

Reply:
added back this at the same place. Will move the check to the place where server config cfg.TSOUpdatePhysicalInterval is initialized